js/load-image.js

```diff
@@ -45,6 +45,9 @@
           // Not using Promises
           if (resolve) resolve(img, data)
           return
+        } else if (img instanceof Error) {
+          reject(img);
+          return;
```

Comment:
@blueimp `reject` was never called here

Reply:
I did not consider exceptions in the transformation call previously, so that's a necessary addition now that exceptions can bubble up through the resoleWrapper as well.